javascript/javascript1/week3/Songs/song.js

```diff
@@ -0,0 +1,78 @@
+const songDatabase = [{
+  songId: 1,
+  title: 'My baby',
+  artist: 'Soggy socks',
+},
+{
+  songId: 2,
+  title: '3 nails in wood',
+  artist: 'The carpenters',
+},
+{
+  songId: 3,
+  title: 'Blacker than black',
+  artist: 'Instant coffee',
+},
+{
+  songId: 4,
+  title: 'When is enough too little?',
+  artist: 'The spies girls',
+},
+];
+
+
+const myPlaylist = [];
+function addToMyPlaylist(index) {
+  let newSong = songDatabase[index];
+  myPlaylist.push(newSong);
+
+
+}
+console.log("My playlist:");
+
+addToMyPlaylist(0);
+addToMyPlaylist(3);
+console.log(myPlaylist);
+
+function addSongToDatabase(song) {
+  songDatabase.push(song);
+}
+const song1 = new Object();
```

Comment:
it's better if you have all function declarations first and then the code that calls them and creates objects at the end of the file